app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetEvents.kt

```diff
@@ -21,3 +21,10 @@ object BloodPressureRemoveClicked : UiEvent {
 }
 
 object BloodPressureDiastolicBackspaceClicked : UiEvent
+
+data class BloodPressureScreenChanged(val type: ScreenType) : UiEvent
+
+enum class ScreenType {
+  BP_ENTRY,
+  DATE_ENTRY
```

Comment:
I am guessing this `enum` will be used elsewhere.
Can we extract it to its own file?

Reply:
I don't think we'll need to use anywhere else. Let's keep this here until we use it elsewhere? What do you think?

Reply:
Yep, just here

Reply:
Update: I was wrong, this needs to be moved out of `BloodPressureEntrySheetEvents`. I'll move it to the screen.